test/unit/3p/test-3p-frame.js

```diff
@@ -105,6 +105,14 @@ describe
           height: 200,
         };
       };
+      div.getAmpDoc = function() {
```

Comment:
Nit: Would an inline object be more concise than a function?

```js
div.getAmpDoc = {
  getMetaByName(name) {
    ...
  }
}
```

Reply:
`.getAmpDoc` needs to be a function (or arrow function) since it is called (i.e. `el.getAmpDoc()`), but if you think it's a worthwhile change to write
```
div.getAmpDoc = function() {
    return {
      getMetaByName(name) {
        const metaTag = document.head.querySelector(`[name="${name}"]`);
        return metaTag ? metaTag.getAttribute('content') : null;
      },
    };
};
```
I can do that.

Reply:
ah right. cleanest would be arrow func, but absolutely no need to make the change.